db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180819232321) do
+ActiveRecord::Schema.define(version: 20180813125658) do
```

Comment:
I was trying to remove db/schema.rb from the repo ... wonder if we need to sync to develop?